contrib/hello-finch/LICENSE

```diff
@@ -0,0 +1,14 @@
+Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Adding the license name (MIT License?) might give more clarity 

(The text is similar to https://choosealicense.com/licenses/mit/ but lacking the sentence of `The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.` ?)

Reply:
Hi!  This is a boilerplate MIT-0 that we use, but honestly since the whole project is otherwise Apache 2.0 I might as well just leave it at that, for simplicity's sake. Thanks!